static/chats.js

```diff
@@ -126,12 +131,14 @@ async function suutiZinju() {
     }
 }
 
-
 function saprotiKomandu(teksts) {
   let vardi = teksts.split(" ");
   let komanda = vardi[0];
   let zinja;
   switch (komanda) {
+    case "/chuck":
```

Comment:
varbūt komandu pārsaucam uz: "/getChuckJoke", vai "/chuckJoke" ?

Reply:
It kā nav kritiski, vienkārši pārējās komandas ir latviski, varētu saglabāt tradīciju.

Reply:
> It kā nav kritiski, vienkārši pārējās komandas ir latviski, varētu saglabāt tradīciju.

Es sataisīju vienkārši /joks